docs/one-click-integrations/github-integration.md

```diff
@@ -14,32 +14,37 @@ Pulse integrates with GitHub Cloud to receive data about changes and deployments
 
 ## Setting up the GitHub integration
 
+!!! note
+    -   You can only install the Pulse GitHub App on an **organization** and not on your personal account.
+    -   [See below](#gh-permissions) the permissions that Pulse requires from your GitHub account.
+
 To set up the GitHub integration:
 
 1.  On Pulse, [expand **Integrations** and select **GitHub**](https://app.pulse.codacy.com/integrations/github){: target="_blank"}.
 
-1.  Click **Install GitHub App** and follow the instructions on the GitHub UI to install the app on your organization.
-
-    !!! important
-        You can only install the Pulse GitHub App on an organization and not on your personal account.
+1.  Click **Install GitHub App** and follow the instructions on the GitHub UI to install the app on your **organization**.
 
     ![Installing the Pulse GitHub App](images/github-installing.png)
 
-1.  Wait until you get a confirmation that Pulse successfully created the integration and the webhook on GitHub.
-
-    ![Pulse GitHub integration set up successfully](images/github-ok.png)
+1.  Confirm that Pulse successfully created the integration and the webhook on GitHub. If there is an error please [contact support](mailto:pulsesupport@codacy.com).
```

Comment:
A couple suggestions here:
* By "integration" are we referring to the GitHub app here? If so, we can reword to clarify that.
* I also added links.

```suggestion
1.  Confirm that Pulse successfully [installed the app](https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/reviewing-your-organizations-installed-integrations) and created the [webhook](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks) on GitHub. If there is an error please [contact support](mailto:pulsesupport@codacy.com).
```

Reply:
Thanks, @nicklem! Excellent feedback. 👍